ccd-definition/AuthorisationCaseEvent/systemUpdate-R2-CUI-nonprod.json

```diff
@@ -299,18 +215,6 @@
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
   },
-  {
-    "CaseTypeID": "CIVIL",
-    "CaseEventID": "CREATE_DASHBOARD_NOTIFICATION_UPLOAD_HEARING_DOCUMENTS_DEFENDANT",
```

Comment:
This one and the below one are supposed to be in the CUI R2 file

Reply:
![image](https://github.com/hmcts/civil-ccd-definition/assets/107135537/5e08c27c-db49-4c02-bc57-722f9e364d18)